### DISCUSSION TITLE
Gizmo Student Processor - File Permission Error

### DISCUSSION THREAD

**Discussion by PunkinMac** [gizmo-platform/gizmo#23] [Q&A]
After successfully writing a code.py file to the student processor, I ran into this problem when I tried to update the same .py again. It also gives me the error to try to load other files. 
![Screenshot 2024-09-11 130321](https://github.com/user-attachments/assets/3cfa582d-7126-4846-aeda-897bb93d0f64)

**Comment by barulicm** [comment]
You will probably have to re-install the CircuitPython firmware or erase the onboard file system through python commands in the Gizmo's REPL as described in [CircuitPython's troubleshooting documentation](https://docs.circuitpython.org/en/latest/docs/troubleshooting.html#file-system-issues). These steps will erase any code saved on the Gizmo, so copy it out to another file on your computer first if you care about it.
